### DISCUSSION TITLE
New commits doesn't appear on contributions charts

### DISCUSSION THREAD

**Discussion by EzequiasAlmeida** [community/community#23236] [New to GitHub]
<p>I’ve just transferred a repo to a company, but, somehow, my new commits to said repo (I still have full access to it) aren’t being shown on my contributions chart. They’re not appearing on Slack notifications either.</p>

**Comment by canuckjacq** [comment]
<p>Have you gone through the steps in this document? It covers just about everything.</p>
<aside class="onebox githubfolder">
  <header class="source">
      <img src="https://docs.github.com/assets/images/site/favicon.svg" class="site-icon" width="16" height="16">
      <a href="https://docs.github.com/en/free-pro-team@latest/github/setting-up-and-managing-your-github-profile/why-are-my-contributions-not-showing-up-on-my-profile#your-local-git-commit-email-isnt-connected-to-your-account" target="_blank" rel="noopener">docs.github.com</a>
  </header>
  <article class="onebox-body">
    

<h3><a href="https://docs.github.com/en/free-pro-team@latest/github/setting-up-and-managing-your-github-profile/why-are-my-contributions-not-showing-up-on-my-profile#your-local-git-commit-email-isnt-connected-to-your-account" target="_blank" rel="noopener">Why are my contributions not showing up on my profile? - GitHub Docs</a></h3>

<p><a href="https://docs.github.com/en/free-pro-team@latest/github/setting-up-and-managing-your-github-profile/why-are-my-contributions-not-showing-up-on-my-profile#your-local-git-commit-email-isnt-connected-to-your-account" target="_blank" rel="noopener">//docs.github.com/en/free-pro-team@latest/github/setting-up-and-managing-your-github-profile/why-are-my-contributions-not-showing-up-on-my-profile</a></p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

**Comment by EzequiasAlmeida** [comment, marked as answer]
<p>My commits were made to a non-default branch. That’s why. When I do the merging they will be accounted for. Thanks.</p>
